pydash/pydash_web/controller/dashboards.py

```diff
@@ -76,58 +54,17 @@ def _dashboard_detail(db):
     :return: A dict structured as the JSON-representation of the given dashboard.
     """
 
-    return _json_mock_dashboard_detail()
-
-def _json_mock_dashboard_detail():
-    from pydash_app.dashboard.dashboard import Dashboard
-    from pydash_app.dashboard.endpoint import Endpoint
-    from pydash_app.dashboard.endpoint_call import EndpointCall
-    import uuid
-    from datetime import datetime, timedelta
-    d = Dashboard("http://foo.io", str(uuid.uuid4()))
-    e1 = Endpoint("foo", True)
-    e2 = Endpoint("bar", True)
-    d.add_endpoint(e1)
-    d.add_endpoint(e2)
-    ec1 = EndpointCall("foo", 0.5, datetime.now(), 0.1, "None", "127.0.0.1")
-    ec2 = EndpointCall("foo", 0.1, datetime.now(), 0.1, "None", "127.0.0.2")
-    ec3 = EndpointCall("bar", 0.2, datetime.now(), 0.1, "None", "127.0.0.1")
-    ec4 = EndpointCall("bar", 0.2, datetime.now() - timedelta(days=1), 0.1, "None", "127.0.0.1")
-    ec5 = EndpointCall("bar", 0.2, datetime.now() - timedelta(days=2), 0.1, "None", "127.0.0.1")
-    ec6 = EndpointCall("bar", 0.2, datetime.now() - timedelta(days=3), 0.1, "None", "127.0.0.2")
-    ec6b = EndpointCall("bar", 0.2, datetime.now() - timedelta(days=3), 0.1, "None", "127.0.0.2")
-    ec6c = EndpointCall("bar", 0.2, datetime.now() - timedelta(days=3), 0.1, "None", "127.0.0.2")
-    ec6d = EndpointCall("bar", 0.2, datetime.now() - timedelta(days=3), 0.1, "None", "127.0.0.2")
-    ec7 = EndpointCall("bar", 0.3, datetime.now() - timedelta(days=5), 0.1, "None", "127.0.0.1")
-    ec8 = EndpointCall("bar", 0.1, datetime.now() - timedelta(days=4), 0.1, "None", "127.0.0.2")
-    d.add_endpoint_call(ec1)
-    d.add_endpoint_call(ec2)
-    d.add_endpoint_call(ec3)
-    d.add_endpoint_call(ec4)
-    d.add_endpoint_call(ec5)
-    d.add_endpoint_call(ec6)
-    d.add_endpoint_call(ec6b)
-    d.add_endpoint_call(ec6c)
-    d.add_endpoint_call(ec6d)
-    d.add_endpoint_call(ec7)
-
-    for n in range(0, 150):
-        d.add_endpoint_call(EndpointCall("foo", n / 10, datetime.now() - timedelta(days = (n % 13)), 0.1, "None", "127.0.0.3"))
-
-    aggregate_data = d.aggregated_data()
-    # d.endpoints['foo'].aggregated_data()
-    # d.endpoints['bar'].aggregated_data()
     def endpoint_dict(endpoint):
         return {
             'name': endpoint.name,
             'aggregates': endpoint.aggregated_data(),
             'enabled': endpoint.is_monitored
         }
 
-    endpoints_dict = [endpoint_dict(endpoint) for endpoint in d.endpoints.values()]
+    endpoints_dict = [endpoint_dict(endpoint) for endpoint in db.endpoints.values()]
```

Comment:
Personally I find 'db' a very confusing name for 'dashboard', because it is very frequently used to mean 'database'. I think just writing out `dashboard` and `dashboards` is more clear here.

Reply:
Again, fair point. Clarity before conciseness.